master_server/master_server.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+
+"""This script works on the asssumption of a certain directory structure.
+
+
+     It then acts the master server and load balances the incoming
```

Comment:
No need for indentation of excessive line spacing here. You can leave one line as a gap.

Reply:
I have made the changes as suggested.